docs/content/flink/procedures.md

```diff
@@ -121,6 +121,27 @@ All available procedures are listed below.
       <td>
          CALL sys.create_tag('default.T', 'my_tag', 10, '1 d')
       </td>
+   </tr>
+    <tr>
+      <td>create_tag_from_timestamp</td>
+      <td>
+         -- Create a tag from the first snapshot that is greater than the specifying timestamp. <br/>
+         CALL [catalog.]sys.create_tag_from_timestamp('identifier', 'tagName', timestamp, fromWatermark, time_retained)
+      </td>
+      <td>
+         To create a tag based on given timestamp. Arguments:
+            <li>identifier: the target table identifier. Cannot be empty.</li>
+            <li>tag: name of the new tag.</li>
+            <li>timestamp (Long): Find the first snapshot greater than this timestamp.</li>
+            <li>fromWatermark (Boolean): Whether to use the snapshot watermark to search.</li>
```

Comment:
maybe just create another procedure? create_tag_from_watermark

Reply:
ok, I will create a new pr to support.